sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/EditContactPersonPage.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.pages.application.contacts;
+
+import org.openqa.selenium.By;
+
+public class EditContactPersonPage {
+  public static final By CONTACT_PERSON_TAB = By.cssSelector("div#tab-contacts-person");
+  public static final By PERSON_UUID = By.cssSelector("#uuid");
```

Comment:
please use the type of element INPUT

Reply:
there's no input there, just plain text in page with the generated UUID which cannot be edited or performed any action

Reply:
done